rc4me/util.py

```diff
@@ -30,32 +30,31 @@ def __init__(self, rc4me_home: Path, rc4me_dest: Path):
         self.dest = rc4me_dest
         # Init rc4me home dir variables (init, prev, current)
         self._init_rc4me_home()
-        # Initialize repo variables
+        # Directory holding source file repo
         self.repo = None
-        self.source = None
-        self.repo_is_local = None
-
-    def set_repo(self, repo: str):
-        """Set repo path to clone."""
-        # Git/local repo from which to clone rc config
-        self.repo = repo
-        self.repo_is_local = Path(self.repo).expanduser().exists()
-        # Init self.source attribute, which will be the path to local repo in
-        # rc4me_home with target config
-        if self.repo_is_local:
-            self.repo = str(Path(self.repo).expanduser())
-            self.source = self.home / Path(self.repo).name
-        else:
-            self.source = self.home / self.repo
 
-    def relink_current_to(self, target: Path):
-        """Change current symlink to point to target path."""
-        # Update prev symlink to point to current
-        self.prev.unlink()
-        self.prev.symlink_to(self.current.resolve())
-        # Update current to point to target
-        self.current.unlink()
-        self.current.symlink_to(target)
+    def _current_is_init(self):
+        """Check if current config is init."""
+        return self.current.resolve() == self.init
+
+    def _unlink_current(self):
+        """Remove symlinks to current config."""
+        if self._current_is_init():
+            return
+        for link, source in self._generate_link_paths():
+            if link.is_symlink() and link.resolve() == source.resolve():
+                logger.info(f"Unlinking {link}")
+                link.unlink()
+
+    def _generate_link_paths(self):
+        """Generate file paths to destination."""
+        for source in self.current.glob("*"):
+            # Skip copying any directories or README files for now (stop-gap)
+            # TODO -- add ability to copy/link directories
+            if source.is_dir() or "README" in source.name:
```

Comment:
Should we add a `forbidden_files` attribute. (`forbidden_files = ["README"]`)

Reply:
All add this as a note to #5